src/ui/library/Select.tsx

```diff
@@ -40,6 +40,7 @@ export function Select({items, onSelect}: Props) {
             title={<Caption>{selectedItem.text ? selectedItem.text : 'Select item'}</Caption>}
             onPress={openMenu}
             right={ItemRight}
+            testID="select-conviction"
```

Comment:
This component is part of our library and it should be generic, `conviction` does not belong here